packages/ckb/src/rgbpp/btc-time.ts

```diff
@@ -80,7 +82,7 @@ export const buildBtcTimeCellsSpentTx = async ({
     lockArgsSet.add(btcTimeCell.output.lock.args);
     const result = await btcAssetsApi.getRgbppSpvProof(
       btcTxIdFromBtcTimeLockArgs(btcTimeCell.output.lock.args),
-      BTC_JUMP_CONFIRMATION_BLOCKS,
+      btcConfirmationBlocks ?? BTC_JUMP_CONFIRMATION_BLOCKS,
```

Comment:
```suggestion
      btcConfirmationBlocks ?? btcTimeCell.output.lock.args.after,
```

Reply:
Maybe the `btcConfirmationBlocks` param is not needed?

Reply:
LGTM

Reply:
https://github.com/ckb-cell/rgbpp-sdk/pull/258/commits/8f99429de45899e5169771e87e73603318a49ae8